ui/apps/platform/src/utils/networkGraphUtils.js

```diff
@@ -1268,3 +1268,24 @@ export function getIsNodeHoverable(type) {
         type === nodeTypes.CIDR_BLOCK
     );
 }
+
+/**
+ * Attempt to get a more descriptive error message based on the text of the server
+ * response, otherwise return a standard error message.
+ *
+ * @param {string} serverError The error message sent from the server.
+ * @param {string=} defaultMessage A default message to use as a fallback for non-specific error cases.
```

Comment:
I have never seen trailing `=` in JSDoc before.

Reply:
Yep, the `=` will mark that parameter as optional:
![image](https://user-images.githubusercontent.com/1292638/165559576-34933620-0329-4357-aa8b-39a8a9e8a540.png)
(works in VSCode at least, not sure about other editors)